spring-cloud-task-docs/src/main/asciidoc/appendix-task-repository-schema.adoc

```diff
@@ -7,3 +7,79 @@ This appendix provides an ERD for the database schema used in the task repositor
 --
 
 image::task_schema.png[]
+
+=== Table Information
+--
+
+.TASK_EXECUTION
+Stores the task execution information.
+[width="80%", cols="1,1,1,1,1,10", options="header"]
+|=========================================================
+|Column Name |Required |Type |Field Length | Default Values |Notes
+
+|TASK_EXECUTION_ID |TRUE |BIGINT | X | X |
+Spring Cloud Task Framework at app startup establishes the next available id as obtained from the `TASK_SEQ`. Or if the record is created outside of task then the value must be populated at record creation time.
+
+|START_TIME |FALSE | DATETIME | X | NULL | Spring Cloud Task Framework at app startup establishes the value.
+
+|END_TIME |FALSE | DATETIME | X | NULL | Spring Cloud Task Framework at app exit establishes the value.
+
+|TASK_NAME |FALSE | VARCHAR | 100 | "" | Spring Cloud Task Framework at app startup will set this to "Application" unless user establish the name using the spring.cloud.task. name as discussed <<features-task-name,here>>
+
+|EXIT_CODE |FALSE | INTEGER | X | 0 | Spring Cloud Task Framework at app exit establishes the value.
+
+|EXIT_MESSAGE |FALSE | VARCHAR | 2500 | "" | Spring Cloud Task Framework at app exit establishes the value.
```

Comment:
This is user defined: https://docs.spring.io/spring-cloud-task/docs/2.0.0.RELEASE/reference/htmlsingle/#features-task-execution-listener-exit-messages.